pilot/pkg/xds/v3/model.go

```diff
@@ -36,7 +36,7 @@ const (
 	// DebugType requests debug info from istio, a secured implementation for istio debug interface.
 	DebugType                 = "istio.io/debug"
 	BootstrapType             = resource.APITypePrefix + "envoy.config.bootstrap.v3.Bootstrap"
-	WorkloadType              = resource.APITypePrefix + "istio.workload.Workload"
+	WorkloadType              = resource.APITypePrefix + "istio.workload.Address"
```

Comment:
I think this should be undone, and we should add a new AddressType and use that instead where appropriate